### PR TITLE
Deny creating tables with column comment if unsupported

### PR DESCRIPTION
**Pull request by ebyhr**
## Description

Deny creating tables with column comment if unsupported

## Documentation

(x) No documentation is needed.

## Release notes

(x) Release notes entries required with the following suggested text:

```markdown
# BigQuery, Cassandra, Delta Lake, Kudu, MariaDB, Memory, MySQL, Oracle, Phoenix, PostgreSQL, Raptor, Redshift, SingleStore, SQL Server, 
* Deny `CREATE TABLE` statement if column comment is specified. ({issue}`12574`)
```

### CODE REVIEW

**Comment by findepi** [comment]
> added the `no-release-notes` label

i think this should be in release notes.
we're changing user-facing aspect of a bunch of connectors.

File: plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java [hunk 1]
```diff
@@ -511,4 +512,11 @@ public RecordCursor cursor(ConnectorTransactionHandle transactionHandle, Connect
             }
         };
     }
+
+    private static void throwIfNotNull(String comment)
```
Comment (findepi) [inline plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java:516]: method signature (`throwIfNotNull(String)`) looks generic, but method behavior is not -- it's for comments only.

i'd inline.

**Review by findepi — APPROVED** [review]

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java [hunk 2]
```diff
@@ -549,6 +549,9 @@ protected String createTableSql(RemoteTableName remoteTableName, List<String> co
 
     protected String getColumnDefinitionSql(ConnectorSession session, ColumnMetadata column, String columnName)
     {
+        if (column.getComment() != null) {
+            throw new TrinoException(NOT_SUPPORTED, "This connector does not support creating tables with column comment");
```
Comment (ssheikin) [inline plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java:553]: @ebyhr `getColumnDefinitionSql` method is used for both 'create table' and 'add column' flows
image
So the exception message during 'add column' flow is not correct.
Could you please help to tackle that situation?

Comment (ebyhr) [reply plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java:553]: Sure, let me fix.

**Pull request merged**